### PR TITLE
adding proxy option

### PR DESCRIPTION
**Pull request by EliranBarDavid**

### CODE REVIEW

File: Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs [hunk 1]
```diff
@@ -76,6 +76,11 @@ public class PerfectoDriver : DriverBase
         [UserConfiguredDescription("Driver Implicit Wait")]
         public int ImplicitWait { get; set; }
 
+        [UserConfigured]
+        [UserConfiguredDefault("")]
+        [UserConfiguredDescription("Proxy Settings,if set, Proxy will be considered otherwiseit will go to system proxy e.g 'genproxy.amdocs.com:8080'")]
```
Comment (MeniKadosh1) [inline Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs]: please fix spaces, and typo in description

File: Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs [hunk 2]
```diff
@@ -177,6 +182,19 @@ private DesiredCapabilities GetWebCapabilities()
 
             capabilities.SetCapability("deviceName", Perfecto_Device_ID);
             capabilities.SetPerfectoLabExecutionId(Perfecto_Host_URL);
+
+            if (!string.IsNullOrEmpty(ProxySettings))
+            {
+                Proxy p = new Proxy();
```
Comment (MeniKadosh1) [inline Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs]: why to have Proxy object creation code duplicated, please set it in function and call it for 2 methods

Comment (EliranBarDavid) [reply Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs]: Done

File: Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs (hunk 1 above)
Comment (MeniKadosh1) [reply Ginger/GingerCore/Drivers/Mobile/Perfecto/PerfectoDriver.cs]: still typo's exist in Description "otherwiseit"

**Comment by MeniKadosh1** [comment]
Merged and deleted branch, please update ticket.

**Pull request merged**